### PR TITLE
Created an example for midi input

### PR DESCRIPTION
**Pull request by StefanSchmelz**
I created a sample that demonstrates midi input.

### CODE REVIEW

File: samples/midi-input/extension/index.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { NodeCG } from "nodecg/types/server";
+import { requireService } from "nodecg-io-core/extension/serviceClientWrapper";
+import { MidiInputServiceClient } from "nodecg-io-midi-input/extension";
+
+module.exports = function (nodecg: NodeCG) {
+    nodecg.log.info("Sample bundle for midi-input started");
+
+    const service = requireService<MidiInputServiceClient>(nodecg, "midi-input");
+    service?.onAvailable((client) => {
+        nodecg.log.info("Client has been updated.");
```
Comment (daniel0611) [inline samples/midi-input/extension/index.ts:10]:
```suggestion
        nodecg.log.info("MIDI input client has been updated.");
```

File: samples/midi-input/extension/index.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        });
+        midiClient.on("mtc", (msg: any) => {
+            printMessage(msg, "mtc");
+        });
+        midiClient.on("select", (msg: any) => {
+            printMessage(msg, "mtc");
+        });
+    });
+
+    service?.onUnavailable(() => nodecg.log.info("Client has been unset."));
```
Comment (daniel0611) [inline samples/midi-input/extension/index.ts:45]:
```suggestion
    service?.onUnavailable(() => nodecg.log.info("MIDI client has been unset."));
```

**Review by daniel0611 — COMMENTED** [review]
Looks good so far!

Can you remove all the implicit `any` types in the handler definitions and then add a linter ignore for the `any` of the `printMessage` function?

Putting the list script here is fine till #109 is solved.

Also please remove the `package-lock.json` file here completely as it is not needed and lerna hoists all dependencies in the repo root.

**Comment by StefanSchmelz** [comment]
Made all changes.

**Comment by daniel0611** [comment]
Thanks @StefanSchmelz

**Pull request merged**